pkg/acquisition/modules/journalctl/journalctl.go

```diff
@@ -0,0 +1,203 @@
+package journalctlacquisition
```

Comment:
maybe `journalctl` is enough and more readable?

Reply:
we decided to keep the long explicit names, but alternative is welcomed